ocaml/idl/datamodel.ml

```diff
@@ -2497,15 +2504,33 @@ let vm_assert_can_migrate = call
     ~name:"assert_can_migrate"
     ~in_product_since:rel_tampa
     ~doc:"Assert whether a VM can be migrated to the specified destination."
+    ~versioned_params:
+      [{param_type=Ref _vm; param_name="vm"; param_doc="The VM"; param_release=tampa_release; param_default=None};
+       {param_type=Map(String,String); param_name="dest"; param_doc="The result of a VM.migrate_receive call."; param_release=tampa_release;  param_default=None};
+       {param_type=Bool; param_name="live"; param_doc="Live migration"; param_release=tampa_release; param_default=None};
+       {param_type=Map (Ref _vdi, Ref _sr); param_name="vdi_map"; param_doc="Map of source VDI to destination SR"; param_release=tampa_release; param_default=None};
+       {param_type=Map (Ref _vif, Ref _network); param_name="vif_map"; param_doc="Map of source VIF to destination network"; param_release=tampa_release; param_default=None};
+       {param_type=Map (String, String); param_name="options"; param_doc="Other parameters"; param_release=tampa_release; param_default=None};
+       {param_type=Map (Ref _vgpu, Ref _gpu_group); param_name="vgpu_map"; param_doc="Map of source vGPU to destination GPU group"; param_release=inverness_release; param_default=Some (VMap [])}
+      ]
+    ~allowed_roles:_R_VM_POWER_ADMIN
+    ~errs:[Api_errors.license_restriction]
+    ()
+
+let vm_assert_can_migrate_sender = call
+    ~name:"assert_can_migrate_sender"
+    ~lifecycle:[]
+    ~doc:"Assertions for VM.assert_can_migrate that must be done on the sending host."
```

Comment:
Just usage of "assertions": `Assertions for VM.assert_can_migrate that must {hold, be verified} on the sending host.`